src/edgeindex.jl

```diff
@@ -16,6 +16,10 @@ function EdgeIndex(g)
     EdgeIndex(iadjl)
 end
 
+@non_differentiable EdgeIndex
```

Comment:
```suggestion
@non_differentiable EdgeIndex(x...)
```
I think you always have to specify the argument

Reply:
Yes, it should be applied on methods, instead of functions